nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -129,16 +128,14 @@ def __init__(
         """
         :param tensor_processor: Backend-specific tensor processor.
         :param aggregation_axes: Axes along which to operate.
-            Registered statistics are stacked along zero axis,
-            axes >=1 correspond to recieved statistic axes shifted left by 1.
```

Comment:
I think we should explain what is going on here explicitly. I mean - you expect aggregation_axes to be axes on a one sample - I, personally, don't expect that. A description `Axes along which to operate.` is not specific enough in my opinion